src/main/resources/db/changelog/init/15062021_create_location_discount_table.yml

```diff
@@ -0,0 +1,27 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1234567890-8
+      author: tesliuk.ua@gmail.com
+      changes:
+        - createTable:
+            schemaName: public
+            tableName: location_discount
+            columns:
+              - column:
+                  name: loc_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+                    primaryKey: true
+                    primaryKeyName: pk_ld_loc_id
+                    foreignKeyName: fk_ld_location
```

Comment:
to improve readability let's use the full names of the tables in fks:
"fk_location_discount_location"